juju/juju
# Budget renaming

## Description of change

> Why is this change needed?

"budgets" are now called "wallets". "allocations" are now called "budgets".

## QA steps

> How do we verify that the change works?

The above change in terminology is applied to subcommand names. However, --budget flags remain unchanged. No regressions.

## Documentation changes

> Does it affect current user workflow? CLI? API?

Yes. Contact @cmars for assistance with documenting this feature.

## Bug reference

> Does this change fix a bug? Please add a link to it.

N/A
